crates/bevy_dev_tools/src/debug_overlay/mod.rs

```diff
@@ -200,7 +200,7 @@ fn outline_roots(
         );
     }
     let window_scale = window.get_single().map_or(1., Window::scale_factor);
-    let scale_factor = window_scale * outline.ui_scale.0;
+    let scale_factor = outline.ui_scale.0;
```

Comment:
We can remove this variable definition if we want.

Reply:
I think it's fine, `scale_factor` is used 2 time on line 243 and 245.